testsuite/integration-arquillian/tests/pom.xml

```diff
@@ -225,6 +225,25 @@
                                 <skip>${auth.server.jboss.skip.unpack}</skip>
                             </configuration>
                         </execution>
+                        <execution>
+                            <id>unpack-undertow-server</id>
```

Comment:
Just to clarify. I've moved this task from base testsuite one level higher (to a parent POM) so that all test modules have access to some required files. This is specifically needed for using [`OAuthClient`](https://github.com/keycloak/keycloak/blob/3eedcdb1542e254a5de74701c6d71fad380fde2a/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/AbstractKeycloakTest.java#L119) in tests (and apparently it was used only in the base testsuite until now).